src/ExportGenerators.mss

```diff
@@ -1027,6 +1027,7 @@ function GenerateNote (nobj) {
 
     ntinfo = ConvertDiatonicPitch(nobj.DiatonicPitch);
     pnum = nobj.Pitch;
+    vel = nobj.OriginalVelocity;
 
```

Comment:
Unless there's a reason to declare this as a variable I'd suggest just doing `libmei.AddAttribute(n, 'vel', nobj.OriginalVelocity);` – easier to see where the value comes from.